third_party/bno055/RPi_BNO055.h

```diff
@@ -274,6 +275,7 @@ class Adafruit_BNO055 : public Adafruit_Sensor {
     void  getCalibration      ( uint8_t* system, uint8_t* gyro, uint8_t* accel, uint8_t* mag);
 
     imu::Vector<3>  getVector ( adafruit_vector_type_t vector_type );
+    bool            getVectors( jcc::Vec3& accel, jcc::Vec3& gyro, jcc::Vec3& mag );
```

Comment:
It's ok to do weird stuff in here if you have to, but you should still prefer to return a struct